src/routes/web.php

```diff
@@ -12,3 +13,4 @@
 Route::get('/profile/show', [UserController::class, 'show'])->name('profile.show');
 Route::post('/logout', [UserController::class, 'logout'])->name('user.logout');
 Route::get('/home', [UserController::class, 'home'])->name('user.home');
+Route::post('/create-post', [PostController::class, 'create']);
```

Comment:
please fix this as per the naming standard format of creating a routes

Reply:
+1 
https://webdevetc.com/blog/laravel-naming-conventions/

Reply:
I have resolved this comment, sirs. Thank you! 🙇